src/module/rolls/preroll-dialog.ts

```diff
@@ -69,11 +75,14 @@ function prerollOptionsWithFormData(
 ): PreRollOptions {
   const opts = cloneDeep(base)
 
-  const valMap: Record<string, string> = form
+  const valMap: Record<string, number> = form
     .serializeArray()
-    .reduce((coll, { name, value }) => ({ ...coll, [name]: value }), {})
+    .reduce(
+      (coll, { name, value }) => ({ ...coll, [name]: parseInt(value, 10) }),
+      {}
+    )
```

Comment:
The use of `parseInt` here breaks the parsing of the checkboxes, which prevents the pre-roll dialog from showing extra challenge dice, automatic outcomes, or preset action dice.

Reply:
ooh, maybe that's the source of the roll graphic breaking a dialog form change. i'll make the function a bit more sensitive to that, then